### ISSUE TITLE
Plugin does not show the conversion option in any workflow tab

### ISSUE THREAD

**Issue by asifdev124** [Vitaliy-1/docxToJats#27]
I have installed and enabled the plugin, uploaded the google docx file, gone through the demo procedure but I am unable to see the xml conversion tab as shown on github page?

**Comment by asifdev124** [comment]
Hi, I am @asifdev124 I have installed and enabled the plugin, uploaded the google docx file, gone through the demo procedure but I am unable to see the xml conversion tab as shown on github page? what I am missing there.. I have reinstalled the plugin, enabled it but no success

**Comment by Vitaliy-1** [comment]
Hi @asifdev124,
What OJS version and DOCX Converter plugin version are working with?

**Comment by asifdev124** [comment]
Both are latest

**Comment by asifdev124** [comment]
I can give you access to the demo site

**Comment by Vitaliy-1** [comment]
Latest meaning OJS 3.3.0-8 and plugin 1.1.1?

**Comment by asifdev124** [comment]
@Vitaliy-1  I am using latest versions.. I can give you access to the site if you wish to look into.. let me know how can I send you securely.

**Comment by asifdev124** [comment]
@Vitaliy-1  I have downloaded yesterday..

**Comment by Vitaliy-1** [comment]
What is the MIME type of file you are trying to convert? It's recorded in the `files` table, `mimetype` column.

**Comment by asifdev124** [comment]
@Vitaliy-1  I have a word document to convert to xml

**Comment by asifdev124** [comment]
@Vitaliy-1 can you send me .tar file and I can reupload to test out?, it actually even does not show the option to convert

**Comment by Vitaliy-1** [comment]
I've encountered situations when the system incorrectly identified MIME type of MS Word files. Can you check how it is recorded in your database?

**Comment by asifdev124** [comment]
@Vitaliy-1 I can't,. I simply created a google docx file and uploaded to test the conversion, but its not showing the convert option...

**Comment by asifdev124** [comment]
@Vitaliy-1 just limited skills

**Comment by Vitaliy-1** [comment]
During OJS installation you have created a database, in which all OJS data is stored. You or the person responsible for it can access the database and try to debug the problem by looking at the table I've pointed above.

**Comment by asifdev124** [comment]
Sorry but no table you have mentioned

**Comment by Vitaliy-1** [comment]
Hmm, `files` table was introduced in OJS 3.3.0. Are you sure that it's the latest version? In older versions, this information was recorded in the `submission_files` table in `file_type` column`. Is it your case?

**Comment by asifdev124** [comment]
@Vitaliy-1  I am absolutely using latest version of ojs and yours available on git.. screenshot attached.
`
![mime](https://user-images.githubusercontent.com/96367599/146769157-1e16c571-0490-47a3-b413-16635b59cd7b.png)
`

**Comment by asifdev124** [comment]
@Vitaliy-1 screenshot attached. if you wanted to take a look, I can create ftp access/ojs access for you.. let me know how can I send you privately, thanks

**Comment by Vitaliy-1** [comment]
Unfortunately, I cannot provide that level of support.

The `mimetype` looks OK to me (at least the part that isn't truncated), But I'm not able to reproduce the problem that you describe with the plugin on the fresh and upgraded OJS 3.3.0-8 instances. Can you describe exactly the steps necessary to reproduce the problem (starting from OJS installation)?

**Comment by asifdev124** [comment]
@Vitaliy-1  I really doing standard, fresh hosting acccount, latest ojs version, latest your plugin version.. created docx file using googledoc, uploaded as an author, passed the stages and trying to see the option which actually convert the document into xml... odd?

**Comment by Vitaliy-1** [comment]
@asifdev124,

So it's not MS Word file but Google Docs file exported as DOCX. It may have a different or changed internal signature and the plugin can't identify DOCX archive structure. I'll take a look.

**Comment by Vitaliy-1** [comment]
Yes, my first assumption seems to be correct, it's problem with MIME type detection. @asifdev124, can you apply this fix to test if it works: https://github.com/Vitaliy-1/docxConverter/commit/f88d160c52fd6d2d3eeae88cd073d39400b0a628?

**Comment by asifdev124** [comment]
@Vitaliy-1 thanks, all is good.

do have any plugin to convert either pdf or xml or docx to html?

**Comment by asifdev124** [comment]
@Vitaliy-1 odd but xml file does not come up.. just loading/spinning the page?

**Comment by Vitaliy-1** [comment]
@asifdev124, Check PHP error log for details

**Comment by asifdev124** [comment]
@Vitaliy-1 It looks like lens plugin has issue, do you recommend any other? I have gone through the issues and most of the users facing same issue.

Also do you have any plugin which convert docx/pdf to html for html galley? 

Thank you
